app/src/main/java/com/infomaniak/drive/utils/DownloadOfflineFileManager.kt

```diff
@@ -170,8 +174,24 @@ class DownloadOfflineFileManager(
             )
         )
 
-        // This line is here to help some devices that don't succeed in automatically creating the file…
-        offlineFile.createNewFile()
+        runCatching {
+            // This line is here to help some devices that don't succeed in automatically creating the file…
+            offlineFile.createNewFile()
+        }.onFailure {
+            Sentry.withScope { scope ->
+                val sentryEvent = SentryEvent().apply {
+                    level = SentryLevel.ERROR
+                    message = Message().apply { this.message = "Failed to create a new file" }
+                    throwable = it
+                }
```

Comment:
Why don't you just use a `Sentry.captureException`?

Reply:
I didn't know SentryLog.e could work with "extras". I changed it